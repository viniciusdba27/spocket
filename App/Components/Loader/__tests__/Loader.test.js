/**
 * @format
 */

import 'react-native'
import React from 'react'
import AppHeader from '../index'
import { Provider } from 'react-redux'
import configureMockStore from 'redux-mock-store'
import renderer from 'react-test-renderer'
import { appLayout } from '../../../../jest/MockedStore'

const mockStore = configureMockStore()
const store = mockStore({ ...appLayout })

jest.useFakeTimers()

it('renders correctly', async () => {
  const component = renderer
    .create(
      <Provider store={store}>
        <AppHeader />
      </Provider>
    )
    .toJSON()
  expect(component).toMatchSnapshot()
})
