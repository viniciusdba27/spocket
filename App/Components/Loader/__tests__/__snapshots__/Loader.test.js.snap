// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "flex": 1,
        "justifyContent": "center",
      },
      Object {
        "bottom": 0,
        "height": 50,
        "left": 0,
        "position": "absolute",
        "width": "100%",
        "zIndex": 100,
      },
    ]
  }
>
  <View
    style={
      Object {
        "backgroundColor": "#ffffff",
        "borderColor": "#e1e1e1",
        "borderRadius": 100,
        "borderWidth": 1,
        "padding": 10,
      }
    }
  >
    <ActivityIndicator
      animating={true}
      color="#999999"
      hidesWhenStopped={true}
      size="large"
    />
  </View>
</View>
`;
